azure-pipelines-ci.yml

```
@@ -26,6 +26,8 @@ resources:
 variables:
   - name: PythonMain
     value: "37"
+  - name: versionHadolint
+    value: "v1.16.3"
```

Let's bump this to the latest if we are introducing it...

```suggestion
    value: "v1.17.6"
```